src/shorah/amplicon.py

```diff
@@ -62,7 +62,8 @@
     if not (diri_exe and b2w_exe):
         # Try fetching diri and b2w exe based on directory structure
         all_dirs = os.path.abspath(__file__).split(os.sep)
-        base_dir = os.sep.join(all_dirs[:all_dirs.index('shorah') + 1])
+        base_dir_idx = [i for i, j in enumerate(all_dirs) if j == 'shorah'][-2]
+        base_dir = os.sep.join(all_dirs[:base_dir_idx + 1])
```

Comment:
I still think it should be `base_dir = os.path.dirname(__file__)`. I understand it doesn't file the executables, but this is an issue in building/compiling the package. I don't think one should expect the executables to be in the `src` directory, as it can be deleted after installation.

Reply:
Doesn't it break if `shorah` appears only one time in `__file__`?

Reply:
yes, indeed

Reply:
No, one can always have a customized path for installation, e.g. using `--prefix`, right?